docs/importing.md

```diff
@@ -0,0 +1,68 @@
+# Importing existing Data into Chronicle
+
+## Importing Data into Chronicle
+
+Chronicle provides a CLI command `import` to load data from a JSON-LD file that
+contains an array of Chronicle Operations. These operations can be used to
+create agents, namespaces, and other resources in the Chronicle database. For
+more information about the data format of the import file, see [Recording Provenance](../recording_provenance/#importing-data-into-chronicle).
+
+### Local development environment
+
+To import data into Chronicle, follow these steps:
+
+- Create a JSON-LD file with an array of Chronicle Operations.
+- Save the file in the Chronicle root directory with a unique name, for example,
+  `import.json`.
+- Run the following command:
+
+  ```bash
+  chronicle --bin chronicle -- import <namespace-id> <namespace-uuid> <json-file-url>
+  ```
+
+If the data is successfully imported, the Chronicle Operations will be added
```

Comment:
Could we maybe DRY this up a bit and remove and/or reference the documentation getting duplicated here in `cli.md`, for example, as well as in `recording_provenance.md`?

Reply:
I've removed the references here, and linked to the CLI docs